timed/tracking/tests/test_report.py

```diff
@@ -242,3 +259,41 @@ def test_absence_update_on_create_report(self):
             Absence.objects.get(pk=absence.pk).duration ==
             employment.worktime_per_day - timedelta(hours=1)
         )
+
+
+class TestReportHypo(TestCase):
+    @given(
+        sampled_from(['csv', 'xlsx', 'ods']),
+        models(
+            Report,
+            comment=characters(blacklist_categories=['Cc']),
+            task=builds(TaskFactory.create),
+            user=builds(UserFactory.create),
+            date=dates(
+                min_date=date(2000, 1, 1),
+                max_date=date(2100, 1, 1),
+            ),
+            duration=timedeltas(
+                min_delta=timedelta(0),
+                max_delta=timedelta(days=1)
+            )
+        )
+    )
+    @settings(timeout=5)
+    def test_report_export(self, file_type, report):
+        User.objects.create_user(username='test', password='1234qwer')
+        client = JSONAPIClient()
+        client.login('test', '1234qwer')
```

Comment:
Hypothesis doesn't support setUp methods properly hence I moved it into its own class which also needs to be subclassed of a special hypothesis TestCase to support transaction per example.

Hence it is needed to create a user here to be able to login.